docs/developing.md

```diff
@@ -189,44 +185,18 @@ To remove all generated artifacts and installed dependencies, run:
 <a name="releasing-minishift"></a>
 ## Releasing Minishift
 
-**Note**: The following paragraphs are partly out of date. Work in progress ...
+<a name="prerequisites-1"></a>
+### Prerequisites
 
-<a name="create-a-release-notes-pr"></a>
-### Create a Release Notes PR
+* A [GitHub personal access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use)
+  defined in your environment as GITHUB_ACCESS_TOKEN
 
-Assemble all the meaningful changes since the last release into the CHANGELOG.md file.
-See [this PR](https://github.com/kubernetes/minikube/pull/164) for an example.
+<a name="cutting-the-release"></a>
+### Cutting the release
 
-<a name="build-and-release-a-new-iso"></a>
-### Build and Release a New ISO
+* Update the [CHANGELOG.md](../CHANGELOG.md) with the changes since the last release.
```

Comment:
@hferentschik , the following lines doesn't have full stop. So annoying me due to inconsistencies :smile: